src/callbacks.cc

```diff
@@ -0,0 +1,82 @@
+#include "../include/callbacks.hh"
+
+alsa_callback::acb::acb() {
+    int errorcode = snd_pcm_open(&handle, "default", SND_PCM_STREAM_CAPTURE, 0);
```

Comment:
does this need to support multiple inputs for the synth and microphone seperately?

Reply:
I'll look into that.

Reply:
So the IC used on the Codec Zero has two input channels, this means the easiest way to make it work is by splitting the channels on the aux in, lets say left is carrier and right is modulator.

I will modify this to create two vectors instead of an interleaved one and then pass both to the callback,

Reply:
Done